### ISSUE TITLE
Hyperparam tuning error for node.toml

### ISSUE THREAD

**Issue by vaibhavsingh007** [Yura52/tabular-dl-revisiting-models#27]
Hi @Yura52 
I noticed you have hyperparam configs (0.toml) for resnet and a few others but not for node. I tried creating a hyperparam tuning config using node and resnet tomls however, ran into the following issue: 

CMD: 
`nohup python bin/tune.py draft/node_screen.toml -f > node_screen.txt & tail -f node_screen.txt`

<img width="1223" alt="image" src="https://user-images.githubusercontent.com/4958034/168333797-f108f723-1b3b-4f21-aee3-1cf114b26aa3.png">

Here's the node toml. Tried commenting the log params but didn't help. 
<img width="502" alt="image" src="https://user-images.githubusercontent.com/4958034/168334065-feb08271-e14c-4d73-8b78-c0561a20fddf.png">

Can you please help?

**Comment by vaibhavsingh007** [comment]
Hi, It works after assigning the intervals. Thanks for the prompt response.

**Comment by vaibhavsingh007** [comment]
Hi @Yura52 , 
Quick question. I was getting an error with certain param combinations and was able to isolate the issue to n_epochs > 5. Have you seen this before or know what could be causing this? I'm not sure this is related to memory or something else is broken. I have 200g mem and 32cores on this machine. Data size is relatively small (62716 x 2761 training set)

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/4958034/171733863-f4e563ef-f74f-4886-a182-5136c3aba865.png">

**Comment by Yura52** [comment]
Hi @vaibhavsingh007 

Unfortunately, the screenshot does not include the main error that is the reason of failure. Could you provide the full traceback?
Also, 62716 x 2761 can be a large dataset, if your algorithm is sensitive to the number of features (not sure to what extent it is true for NODE).

**Comment by vaibhavsingh007** [comment]
Hi @Yura52 , 
I wish I had more info but that actually is the full error. Please take a look: 

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/4958034/171858287-6b160f18-fec5-4de7-8566-048671e8be3f.png">

**Comment by vaibhavsingh007** [comment]
Gotcha! I'm using it for benchmarking as of now. So far the performance I've got is from FT Transformer. Thanks.

**Comment by manmeet3591** [comment]
Hi @Yura52 can you please help me with this error ? I am trying to do hyperparamter tuning with node with the following toml file 
program = 'bin/node.py'
  
[base_config]
seed = 0

    [base_config.data]
    normalization = 'quantile'
    path = 'data/bathymetry'
    y_policy = 'mean_std'

    [base_config.model]
    tree_dim = 3
    choice_function = "entmax15"
    bin_function = "entmoid15"


    [base_config.training]
    batch_size = 1024
    eval_batch_size = 8192
    n_epochs = 10000
    optimizer = 'adamw'
    patience = 20

[optimization.options]
n_trials = 10000

[optimization.sampler]
seed = 0

[optimization.space.model]
num_layers = ['int', 2, 10]
layer_dim = ['int', 256, 2048]
depth = ['int', 4, 10]


[optimization.space.training]
lr = [ 'loguniform', 1e-05, 0.01 ]
weight_decay = [ '?loguniform', 0.0, 1e-06, 0.001 ]
batch_size = ['int', 64, 1024]
n_epochs = ['int', 2,10]
patience = ['int', 5, 25]



![node](https://user-images.githubusercontent.com/8520638/195356972-8585ad46-eac0-4346-a97b-6f3093dd18c6.png)

**Comment by manmeet3591** [comment]
Thank you. I was able to fix the error.

**Comment by Yura52** [comment]
@manmeet3591 I missed your question, sorry for that, glad to hear that you fixed the issue
